Improve worker queues overview, other minor improvements (#1411)
astronomer/docs: astro/configure-deployment-resources.md

---
sidebar_label: 'Configure a Deployment'
title: 'Configure a Deployment'
id: configure-deployment-resources
description: Learn how to configure Astro Deployment resources.
---

> After you create an Astro Deployment, you can modify its settings to fit the needs your organization.

Reviewer: ```suggestion
After you create an Astro Deployment, you can modify its settings to meet the unique requirements of your organization.
```

On Astro, you can:

- Set resources for the Airflow scheduler and workers.
- Update a Deployment name and description.
- Add or delete a Deployment alert email.
- Transfer a Deployment from one Workspace in your Organization to another.
- Delete a Deployment.

## Set Deployment resources

To ensure that your tasks have the CPU and memory required to complete successfully, you can set resources for:

- The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html), which is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met.
- Workers, which are responsible for executing tasks that have been scheduled and queued by the scheduler.

> Worker and scheduler resources must be set per Deployment and are managed separately from Cluster-level infrastructure. Any additional components that Astro requires, including PgBouncer, KEDA, and the triggerer, are managed by Astronomer.

Reviewer: ```suggestion
Worker and scheduler resources must be set for each Deployment and are managed separately from Cluster-level infrastructure. Any additional components that Astro requires, including PgBouncer, KEDA, and the triggerer, are managed by Astronomer.
```

### Worker queues 

> Worker queues are a set of configurations that apply to a group of workers in your Deployment. Each Deployment includes a required `default` worker queue for running tasks, but you can configure additional worker queues to define CPU and memory limits for your tasks.

Reviewer: ```suggestion
Worker queues are a set of configurations that apply to a group of workers in your Deployment. Each Deployment includes a `default` worker queue for running tasks, but you can configure additional worker queues to define CPU and memory limits for your tasks.
```

---
sidebar_label: 'Configure a Deployment'
title: 'Configure a Deployment'
id: configure-deployment-resources

> description: Learn how to configure Astro Deployment resources.

Reviewer: ```suggestion

```

  <meta name="og:description" content="Modify the resource settings of a Deployment to make sure that your tasks have the CPU and memory required to complete successfully." />
</head>

After you create an Astro Deployment, you can modify its settings to meet the unique requirements of your organization. On Astro, you can:

- Allocate resources for the Airflow scheduler and workers.
- Update a Deployment name and description.
- Add or delete a Deployment alert email.
- Transfer a Deployment from one Workspace in your Organization to another.
- Delete a Deployment.

## Set Deployment resources

To ensure that your tasks have the CPU and memory required to complete successfully, you can set resources for:

- The [Airflow scheduler](https://airflow.apache.org/docs/apache-airflow/stable/concepts/scheduler.html), which is responsible for monitoring task execution and triggering downstream tasks when the dependencies are met.
- Workers, which are responsible for executing tasks that have been scheduled and queued by the scheduler.

> Worker and scheduler resources must be set for each Deployment and are managed separately from Cluster-level infrastructure. Any additional components that Astro requires, including PgBouncer, KEDA, and the triggerer, are managed by Astronomer.

Author: ```suggestion
Worker and scheduler resources must be set for each Deployment and are managed separately from cluster-level infrastructure. Any additional components that Astro requires, including PgBouncer, KEDA, and the triggerer, are managed by Astronomer.
```

    - Click **Delete** next to the email you want to delete.
    - Click **Yes, Continue**.

> In addition to alert emails for your Deployments, Astronomer recommends subscribing to the [Astro status page](https://status.astronomer.io). When you subscribe, you get alerted to system-wide incidents by email in real time.

Reviewer: ```suggestion
In addition to alert emails for your Deployments, Astronomer recommends subscribing to the [Astro status page](https://status.astronomer.io). When you subscribe, you'll receive email notifications about system-wide incidents in real time.
```
